src/lib.rs

```diff
@@ -1260,6 +1264,30 @@ mod test {
         Ok(())
     }
 
+    #[cfg(feature = "polars")]
+    #[test]
+    fn test_query_polars_small() -> Result<()> {
+        use crate::polars_dataframe::PolarsDataFrame;
+        use polars::prelude::{df, NamedFrom};
+
+        let db = checked_memory_handle();
+        let sql = "BEGIN TRANSACTION;
+                   CREATE TABLE test(t INTEGER);
+                   INSERT INTO test VALUES (1); INSERT INTO test VALUES (2); INSERT INTO test VALUES (3); INSERT INTO test VALUES (4); INSERT INTO test VALUES (5);
+                   END TRANSACTION;";
+        db.execute_batch(sql)?;
+        let mut stmt = db.prepare("select t from test order by t desc")?;
+        let actual = stmt.query_polars([])?.collect::<PolarsDataFrame>().into_inner();
```

Comment:
` let actual: Vec<DataFrame> = db.prepare("select t from test order by t")?.query_polars([])?.collect();`

So we don't need `PolarsDataFrame`?

Reply:
You review [this](https://github.com/wangfenjin/duckdb-rs/pull/169#discussion_r1209659636) then I will modify.